### PR TITLE
chg ! demoapp

### PR DESCRIPTION
**Pull request by vitali-yanushchyk-valor**

### CODE REVIEW

File: compose.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@ x-common: &common
     target: python_dev_deps
   platform: linux/amd64
   environment:
-    - DEBUG=true
```
Comment (domdinicola) [inline compose.yml:8]: pls remove

File: compose.yml [hunk 2]
```diff
@@ -18,6 +18,8 @@ x-common: &common
     - FILE_STORAGE_DEFAULT=django.core.files.storage.FileSystemStorage
     - FILE_STORAGE_DNN=storages.backends.azure_storage.AzureStorage?azure_container=dnn&overwrite_files=True&connection_string=DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://azurite:10000/devstoreaccount1;
     - FILE_STORAGE_HOPE=storages.backends.azure_storage.AzureStorage?azure_container=hope&overwrite_files=True&connection_string=DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://azurite:10000/devstoreaccount1;
+    - FILE_STORAGE_MEDIA=storages.backends.azure_storage.AzureStorage?azure_container=media&overwrite_files=True&connection_string=DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://azurite:10000/devstoreaccount1;
```
Comment (domdinicola) [inline compose.yml:21]: should we need to store credentials?

File: tests/extras/demoapp/compose.yml [hunk 3]
```diff
@@ -14,6 +14,8 @@ x-common: &common
     - FILE_STORAGE_DEFAULT=django.core.files.storage.FileSystemStorage?location=/var/hope_dedupe_engine/default
     - FILE_STORAGE_DNN=storages.backends.azure_storage.AzureStorage?azure_container=dnn&overwrite_files=True&connection_string=DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://azurite:10000/devstoreaccount1;
     - FILE_STORAGE_HOPE=storages.backends.azure_storage.AzureStorage?azure_container=hope&overwrite_files=True&connection_string=DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://azurite:10000/devstoreaccount1;
+    - FILE_STORAGE_MEDIA=storages.backends.azure_storage.AzureStorage?azure_container=media&overwrite_files=True&connection_string=DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://azurite:10000/devstoreaccount1;
```
Comment (domdinicola) [inline tests/extras/demoapp/compose.yml:17]: same ^

File: compose.yml (hunk 1 above)
Comment (vitali-yanushchyk-valor) [reply compose.yml:8]: This compose.yml should be using only for local development purposes.
btw, this parameter was commented, but I can remove it.

File: compose.yml (hunk 2 above)
Comment (vitali-yanushchyk-valor) [reply compose.yml:21]: These are the credentials for Azurite and are public:
https://learn.microsoft.com/en-us/azure/storage/common/storage-use-azurite?tabs=visual-studio%2Cblob-storage#http-connection-strings

File: compose.yml (hunk 1 above)
Comment (domdinicola) [reply compose.yml:8]: if it's for development let's put true then

**Comment by domdinicola** [comment]
@vitali-yanushchyk-valor pls fix conflicts

**Comment by vitali-yanushchyk-valor** [comment]
> @vitali-yanushchyk-valor pls fix conflicts

done

**Pull request merged**